packages/@aws-cdk/aws-neptune/test/endpoint.test.ts

```diff
@@ -0,0 +1,29 @@
+import { Endpoint } from '../lib';
+
+// A numeric CDK token (see: https://docs.aws.amazon.com/cdk/latest/guide/tokens.html#tokens_number)
+const CDK_NUMERIC_TOKEN = -1.8881545897087626e+289;
+
+describe('Endpoint', () => {
+  test('accepts tokens for the port value', () => {
```

Comment:
_If_ we want to keep this test, I think the right approach would be to use `Token.asAny`, and verifying that the port is an unresolved token.